src/engine/strat_engine/tests/util.rs

```diff
@@ -0,0 +1,51 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+use std::env;
+use std::path::PathBuf;
+use std::process::Command;
+
+use nix::mount::{umount};
+use mnt::{get_submounts};
+
+pub fn dm_remove_all() -> () {
+    let output = Command::new("/usr/sbin/dmsetup")
+                     .arg("remove_all")
+                     .arg("--force")  // TODO: Should we be using force?
+                     .output()
+                     .expect("failed to execute process dmsetup");
+
+    if !output.status.success() {
+        eprintln!("dmsetup status: {}", output.status);
+        eprintln!("dmsetup stdout: {}", String::from_utf8_lossy(&output.stdout));
+        eprintln!("dmsetup stderr: {}", String::from_utf8_lossy(&output.stderr));
+    }
+}
+
+pub fn clean_up() -> () {
+
+    // To allow debugging, especially when running individual test cases lets offer an option
+    // to not clean up
+    if env::var("STRATIS_TEST_SKIP_CLEAN_UP").is_ok() {
+        eprintln!("Skipping clean up as requested");
+        return;
+    }
+
+    // Do our best to clean up after a unit test
```

Comment:
If you had a precise header comment you could remove this comment, and state precisely what "cleanup" means in the header comment.